### ISSUE TITLE
 Composer broken on NodeBB 1.16.2 (`plugins may not be compatible`)

### ISSUE THREAD

**Issue by LM1LC3N7** [NodeBB/nodebb-plugin-composer-quill#176]
Hello,

Since I have installed this plugin, no more composer are showing up when I want to create a new comment on the forum.

![image](https://user-images.githubusercontent.com/14235706/114871613-bee49100-9df9-11eb-8356-5a84661485cc.png)

About the error "Missing translation", I tried to set `en-US` and `fr-FR` and got the same error.

**Start logs on the server side:**

```bash
2021-04-15T12:46:26.613Z [4567/574] - info: [app] Shutdown complete.
[cluster] Child Process (574) has exited (code: 0, signal: null)

NodeBB v1.16.2 Copyright (C) 2013-2014 NodeBB Inc.
This program comes with ABSOLUTELY NO WARRANTY.
This is free software, and you are welcome to redistribute it under certain conditions.
For the full license, please visit: http://www.gnu.org/copyleft/gpl.html

Clustering enabled: Spinning up 1 process(es).

2021-04-15T12:46:27.258Z [4567/782] - info: Initializing NodeBB v1.16.2 http://forum.tld
2021-04-15T12:46:28.438Z [4567/782] - info: [socket.io] Restricting access to origin: http://forum.tld:*
(node:782) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
2021-04-15T12:46:28.828Z [4567/782] - warn: [plugins] File not found: /nodebb/node_modules/nodebb-plugin-composer-default/static/lib/composer/scheduler.js (Ignoring)
Thu, 15 Apr 2021 12:46:28 GMT node-telegram-bot-api deprecated Automatic enabling of cancellation of promises is deprecated.
In the future, you will have to enable it yourself.
See https://github.com/yagop/node-telegram-bot-api/issues/319. at internal/modules/cjs/loader.js:1063:30

2021-04-15T12:46:28.971Z [4567/782] - warn: [plugins/load] The following plugins may not be compatible with your version of NodeBB. This may cause unintended behaviour or crashing. In the event of an unresponsive NodeBB caused by this plugin, run `./nodebb reset -p PLUGINNAME` to disable it.
  * nodebb-plugin-composer-quill

2021-04-15T12:46:28.986Z [4567/782] - info: [telegram-notification] Starting bot. Please DO NOT do anythig.
2021-04-15T12:46:31.117Z [4567/782] - info: [api] Adding 0 route(s) to `api/v3/plugins`
2021-04-15T12:46:31.129Z [4567/782] - info: Routes added
2021-04-15T12:46:31.133Z [4567/782] - info: NodeBB Ready
2021-04-15T12:46:31.134Z [4567/782] - info: Enabling 'trust proxy'
2021-04-15T12:46:31.136Z [4567/782] - info: NodeBB is now listening on: 0.0.0.0:4567
```

**Details:** 
* NodeBB v1.16.2

**Plugins:**

```bash
Active plugins:
        * nodebb-plugin-announcements@0.1.10 (installed, enabled)
        * nodebb-plugin-category-notifications@3.0.3 (installed, enabled)
        * nodebb-plugin-category-optin@2.0.0 (installed, enabled)
        * nodebb-plugin-composer-default@6.5.5 (installed, disabled)
        * nodebb-plugin-composer-quill@2.1.0 (installed, enabled)
        * nodebb-plugin-dbsearch@4.1.2 (installed, enabled)
        * nodebb-plugin-embed@3.0.19 (installed, enabled)
        * nodebb-plugin-emoji@3.5.1 (installed, enabled)
        * nodebb-plugin-emoji-android@2.0.0 (installed, disabled)
        * nodebb-plugin-emoji-one@2.0.5 (installed, enabled)
        * nodebb-plugin-lastlife-registration-question@0.1.0 (installed, enabled)
        * nodebb-plugin-markdown@8.12.4 (installed, disabled)
        * nodebb-plugin-mentions@2.13.6 (installed, enabled)
        * nodebb-plugin-private-forum@1.2.0 (installed, enabled)
        * nodebb-plugin-question-and-answer@0.8.7 (installed, disabled)
        * nodebb-plugin-registration-notification@1.2.5 (installed, enabled)
        * nodebb-plugin-soundpack-default@1.0.0 (installed, disabled)
        * nodebb-plugin-spam-be-gone@0.7.7 (installed, disabled)
        * nodebb-plugin-telegram-notifications-plus@1.0.4 (installed, enabled)
        * nodebb-rewards-essentials@0.1.4 (installed, enabled)
        * nodebb-theme-lavender@5.0.17 (installed, disabled)
        * nodebb-theme-persona@10.3.19 (installed, enabled)
        * nodebb-theme-slick@1.3.8 (installed, disabled)
        * nodebb-theme-vanilla@11.3.10 (installed, disabled)
        * nodebb-widget-essentials@5.0.2 (installed, enabled)
        * nodebb-widget-search-bar@1.1.2 (installed, enabled)
```

**Comment by barisusakli** [comment]
NodeBB 1.16.2 doesn't have the `hooks` module, you need downgrade the quill plugin or update your nodebb to the latest 1.17.0 beta

**Comment by LM1LC3N7** [comment]
Thanks for the answer :)
Do you know why it was removed in 1.16.2 and added again in 1.17.0?

**Comment by barisusakli** [comment]
`hooks` module was added in 1.17.0 it wasn't available before.

**Comment by LM1LC3N7** [comment]
> you need downgrade the quill plugin 
What it the last compatible version?

**Comment by barisusakli** [comment]
Use 1.8.0 if you are on nodebb 1.16.2

**Comment by julianlam** [comment]
Hi @LM1LC3N7 sorry about this regression. It seems I've neglected to update the compatibility string when I added the logic using the client-side hooks module.

Will update the versioning shortly.

**Comment by julianlam** [comment]
@LM1LC3N7 v2.1.1 is published, but it is exactly the same as v2.1.0. You should not have been offered v2.1.0 by NodeBB since it is not compatible. The version string was actually fine.
